client/src/pages/JobsSearch/index.tsx

```diff
@@ -85,7 +85,7 @@ function JobsSearch() {
   } else {
     tabChild = (
       <>
-        {jobs.map((job) => (<JobCard job={job} key={job.title} id={job.id} />))}
+        {jobs.map((job) => (<JobCard userRole="user" job={job} key={job.title} id={job.id} />))}
```

Comment:
```suggestion
        {jobs.map((job) => (<JobCard userRole="user" job={job} key={job.title} id={job.id} >
        <Accordion disabled={!proposals.length}>other stuff</Accordin>
        </JobCard>)
        
        )}
```
and in other places may we pass children may not,
This is a general way to make the component reusable, passing children would be more helpful instead of rendering different components conditionally